### PR TITLE
session 18 notes

### PR DESCRIPTION
**Pull request by TiaMcDowell**

### CODE REVIEW

File: docs/notes/Session Notes/Session 18.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Session 18
+#session
+Prev: [[Session 17]]
+Next: [[Session 19]]
+
+[[Rictavio]] entered borovia 34 days ago. Party entered 7 days ago. [[Rictavio]] shows us his saber tooth lion. [[Daedi]] sees its on strings. Served it warefolve meat. Was warned to stay away from the abby. He would preform arround noon.
```
Comment (aldman123) [inline docs/notes/Session Notes/Session 18.md]: Typo in warefolve

File: docs/notes/Session Notes/Session 19.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- party goes to the tower [[Lake Baratok]]
+- [[Daedi]] finds a head of a [[Vastani]] man in a locked chest on the 4th floor
+- [[Rudolph van Richten]] is raciest against [[Vastani]] and has kept thier heads arround for information
+- in [[Ezmerelda d'Avenir]]  there is a note from [[Rudolph van Richten]]![[Pasted image 20220821195642.png]]
+- [[Ezmerelda d'Avenir]] has left borovia before, the [[Vastani]] can freely
+- usually people are lured into [[Borovia(valley)]] by [[Vastani]]
+- many of the [[Valaki]] people are outside of [[Krezk]] begging for enterence becuase of [[Strahd]]s attak on [[Valaki]]
+- part of [[Strahd]]s curse, his land is full of the dull(souless)
+- party talks to the burgemaster(demitri)
+- Party talks to the burgelaster, he doesnt like us very much because we insulted him, but aggreed to allow people into [[Krezk]]
```
Comment (aldman123) [inline docs/notes/Session Notes/Session 19.md]: You also blackmailed him!

File: docs/notes/Session Notes/Session 18.md (hunk 1 above)
Comment (TiaMcDowell) [reply docs/notes/Session Notes/Session 18.md]: I dont know what you mean :(

File: docs/notes/Session Notes/Session 19.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- Party goes to the tower [[Lake Baratok]]
+- [[Daedi]] finds a head of a [[Vastani]] man in a locked chest on the 4th floor
+- [[Rudolph van Richten]] is raciest against [[Vastani]] and has kept their heads around for information
+- in [[Ezmerelda d'Avenir]] there is a note from [[Rudolph van Richten]]![[Pasted image 20220821195642.png]]
+- [[Ezmerelda d'Avenir]] has left [[Borovia(valley)]] before, the [[Vastani]] can freely
+- usually people are lured into [[Borovia(valley)]] by [[Vastani]]
+- many of the [[Valaki]] people are outside [[Krezk]] begging for entrance because of [[Strahd]]s attack on [[Valaki]]
+- part of [[Strahd]]s curse, his land is full of the dull (souless)
+- party talks to the [[Burgomaster Demetri]]
+- Party talks to [[Burgomaster Demetri]], he doesn't like us very much because we insulted and balck mailed him, but agreed to allow people into [[Krezk]]
```
Comment (aldman123) [inline docs/notes/Session Notes/Session 19.md]: Typo in balck

**Review by aldman123 — APPROVED** [review]

File: docs/notes/Session Notes/Session 19.md (hunk 3 above)
Comment (aldman123) [reply docs/notes/Session Notes/Session 19.md]: I even spelt it for you in my previous comment

**Pull request merged**